packages/cli/tests/e2e/apim/BotAddApim.tests.ts

```diff
@@ -37,7 +37,7 @@ describe("Configuration successfully changed when with different plugins", funct
 
   it(`bot + apim`, { testPlanCaseId: 15685003 }, async function () {
     if (isV3Enabled()) {
-      this.skip();
+      return;
```

Comment:
Can you delete these cases directly? Seems we don't need these V4 cases any more.

cc @LongOddCode in case you have additional comments.